providers/src/airflow/providers/standard/sensors/date_time.py

```diff
@@ -69,7 +69,7 @@ class DateTimeSensor(BaseSensorOperator):
 
             DateTimeSensor(
                 task_id="wait_for_0100",
-                target_time="{{ next_execution_date.tomorrow().replace(hour=1) }}",
+                target_time="{{ next_logical_date.tomorrow().replace(hour=1) }}",
```

Comment:
We should also remove `next_logical_date`. Since this is just an example, we can change it to use `data_interval_end` instead.

Reply:
Changed it